src/seedu/addressbook/data/AddressBook.java

```diff
@@ -143,4 +143,6 @@ public UniquePersonList getAllPersons() {
     public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
+    
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.